engine/src/lin_time.cpp

```diff
@@ -113,15 +113,22 @@ void setTimeCompression( float tc )
 bool toggle_pause()
 {
     static bool paused = false;
+    fprintf(stderr, "toggle_pause() called in lin_time.cpp\n");
```

Comment:
fine with me; do we wnat to send the to the Boost Log?

Reply:
@BenjamenMeyer Yes. Duh. All I had to do to make that work was `#include "vsfiilesystem.h"`. I was forgetting.

Do you think we need to run the play tests again?

Reply:
Wouldn't hurt, but probably doesn't need to be as extensive